packages/runtime/example/module/index.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>meta-ui runtime example: nested components</title>
+  </head>
+  <body>
+    <div id="root"></div>
+    <script type="module">
+      import renderApp, { registry } from '../../src/main.tsx';
+      import { Type } from '@sinclair/typebox';
+      const exampleModule = {
+        version: 'core/v1',
+        kind: 'Module',
+        parsedVersion: {
+          category: 'core/v1',
+          value: 'littleForm',
+        },
+        metadata: {
+          name: 'littleForm',
+        },
+        spec: {
+          properties: {
+            btnText: Type.String(),
+          },
+          events: ['onSubmit'],
+          stateMap: {
+            value: '{{$moduleId}}__input.value',
+          },
+        },
+        components: [
+          {
+            id: '{{$moduleId}}__hstack',
```

Comment:
Adding `moduleId` as a prefix to avoid components id conflictions looks reasonable, but writing the boilerplate many times maybe a little boring for end users.

Not sure whether we should add this automatically and let the user only write `id: 'hstack'` or not, but it's out of this PR's scope so we can discuss about it in the coming weekly meeting.